testsuite/CHANGELOG

```diff
@@ -17,6 +17,7 @@ and https://github.com/MDAnalysis/mdanalysis/wiki/UnitTests
          tyler.je.reddy
 
   * 0.16
+    - Added unit tests for MDAnalysis.analysis.psa exception handling
```

Comment:
The CHANGELOG is mostly for changes noticeable to users, so we don't really need a log of every test added, plus it's causing merge conflicts too.